ci_pod/jupyterlab/Dockerfile

```diff
@@ -4,18 +4,36 @@ ARG JUPYTERLAB_VERSION
 ENV JUPYTERLAB_VERSION=$JUPYTERLAB_VERSION
 ARG MAINTAINER
 ENV MAINTAINER=$MAINTAINER
-
 LABEL maintainer="Your Name <$MAINTAINER>"
+ARG OC_VERSION
+ENV OC_VERSION=$OC_VERSION
+ARG VIRTCTL_VERSION
+ENV VIRTCTL_VERSION=$VIRTCTL_VERSION
 
 # Install system dependencies
 RUN dnf update -y && \
     dnf install -y wget podman curl python3 python3-pip python3-devel nodejs npm && \
     dnf clean all && \
     rm -rf /var/cache/dnf && \
-    pip3 install --no-cache-dir jupyterlab=="$JUPYTERLAB_VERSION" benchmark-runner && \
+    pip3 install --no-cache-dir jupyterlab=="$JUPYTERLAB_VERSION" benchmark-runner bokeh && \
     jupyter labextension install @jupyter-widgets/jupyterlab-manager
 
+# install oc/kubectl client tools for OpenShift/Kubernetes
+RUN  curl -L https://github.com/openshift/okd/releases/download/$OC_VERSION/openshift-client-linux-$OC_VERSION.tar.gz -o  ~/openshift-client-linux-$OC_VERSION.tar.gz \
+     && tar -xzvf  ~/openshift-client-linux-$OC_VERSION.tar.gz -C  ~/ \
+     && rm -rf ~/openshift-client-linux-$OC_VERSION.tar.gz \
```

Comment:
These are individual files; you should use `rm -f` rather than `rm -rf`.  `rm -rf` works, but it states clearly that you intend to remove a single file, not a directory.

Reply:
fixed